src/twisted/web/test/test_http.py

```diff
@@ -2932,6 +2932,19 @@ def test_finishAfterConnectionLost(self):
         self.assertRaises(RuntimeError, req.finish)
 
 
+    def test_writeAfterConnectionLost(self):
+        """
+        Calling L{Request.write} after L{Request.connectionLost} has been
+        called should not throw an exception. L{RuntimeError} will be raised
```

Comment:
Twisted docstring style guide says that this should be "... after L{Request.connectionLost} has been called *does not* throw an exception.  L{RuntimeError} *is raised ...".  Also, consistent use of "raise" (instead of "throw") would be good.

Reply:
language changed in https://github.com/twisted/twisted/pull/1106/commits/00a59794e8f3f6d9e596c3c600ef6c0dba3da084